apps/ledger-live-desktop/src/storyly/StorylyProvider.tsx

```diff
@@ -46,6 +50,8 @@ const StorylyProvider: React.FC<StorylyProviderProps> = ({ children }) => {
       token: token,
     });
 
+    storylyRef.current?.setLang({ language });
```

Comment:
Why is this one needed if `setLang` is already called in the other `useEffect` ?

Reply:
fixed